synthtool/languages/node.py

```diff
@@ -42,6 +44,39 @@ def read_metadata():
         return data
 
 
+def template_metadata() -> Dict[str, Any]:
+    """Load node specific template metadata.
+
+    Returns:
+        Dictionary of metadata. Includes the entire parsed contents of the package.json file if
+        present. Other expected fields:
+        * quickstart (str): Contents of the quickstart snippet if available, otherwise, ""
+        * samples (List[Dict[str, str]]): List of available samples. See synthtool.gcp.samples.all_samples()
+    """
+    metadata = {}
+    try:
+        metadata = read_metadata()
+    except FileNotFoundError:
+        pass
+
+    all_samples = samples.all_samples(["samples/*.js"])
```

Comment:
I think we end up adding the prefix `samples` twice somewhere around here.

Reply:
Missed one in the `samples/README.md` template